brainzutils/cache.py

```diff
@@ -28,22 +32,37 @@
 ENCODING_ASCII = "ascii"
 
 
-def init(host="localhost", port=6379, db_number=0, namespace=""):
+def init(host: str = "localhost", port: int = 6379, db_number: int = 0,
+         namespace: str = "", client_name: Optional[str] = None):
     """Initializes Redis client. Needs to be called before use.
 
     Namespace versions are stored in a local directory.
 
     Args:
-        host (str): Redis server hostname.
-        port (int): Redis port.
-        db_number (int): Redis database number.
-        namespace (str): Global namespace that will be prepended to all keys.
+        host: Redis server hostname.
+        port: Redis port.
+        db_number: Redis database number.
+        namespace: Global namespace that will be prepended to all keys.
+        client_name: The client name to assign to the redis connection. usually
+         only needs to be specified if at all during testing or development.
```

Comment:
not sure exactly how to make this clearer. I want to say that setting this doesn't affect the behaviour of the client, but that we should do it because it's useful for debugging.

```suggestion
        client_name: The client name to assign to the redis connection. This value is used to identify which clients
          are connected to a server, and is only used for debugging purposes.
```